tests/test_utils.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import numpy as np
+import pyfar as pf
+
+from imkar import utils
+
+
+@pytest.mark.parametrize("c_value",  [
+    (0), (0.2), (0.5), (0.8), (1)])
+@pytest.mark.parametrize("frequencies",  [
+    ([100, 200]), ([100])])
+def test_random_constant_coefficient(
```

Comment:
I would suggest that all tests start with `test_paris_formula_` in case more functios are added to the module later.